src/deck/guides/3.0-upgrade.md

```diff
@@ -0,0 +1,85 @@
+---
+title: Upgrade to Kong Gateway 3.x
+---
+
+Starting with {{site.base_gateway}} 3.0, paths using regular expressions must have a `~` prefix.
+The decK format version must be `3.0` to work with the new paths.
+
+If you [migrate your {{site.base_gateway}} database](/gateway/latest/upgrade/) from 2.8.x to 3.0, the paths in the database will have the `~` prefix added.
+This can cause configuration drift between state files and the database.
```

Comment:
```suggestion
This causes configuration drift because the routes that exist in the database, and the ones in your configuration file will use a different format.
```